views/editor/warnings.html

```diff
@@ -4,7 +4,7 @@
   <div class="message-box">
     <h1>{{ gettext("errorUnsupportedBrowser") }}</h1>
     <p>
-      {{ gettext("errorUnsupportedBrowserSuggestion") | instantiate | safe }}
+      {{ gettext("errorUnsupportedBrowserSuggestion1") | instantiate | safe }}
```

Comment:
since we don't have any more template parameters inside the strings (previously was `{{ locale }}`), we should remove the `instantiate` filter here so that it's just `{{ gettext("errorUnsupportedBrowserSuggestion1") | safe }}`